File: src/main/java/upbrella/be/user/dto/token/OauthToken.java
package upbrella.be.user.dto.token;

import com.fasterxml.jackson.databind.PropertyNamingStrategies;
import com.fasterxml.jackson.databind.annotation.JsonNaming;
import lombok.*;

@Getter
@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class OauthToken {

    private String accessToken;
    private String refreshToken;
    private String tokenType;
    private Long expiresIn;
}

File: src/main/kotlin/upbrella/be/user/dto/token/OauthToken.kt
package upbrella.be.user.dto.token

import com.fasterxml.jackson.databind.PropertyNamingStrategies
import com.fasterxml.jackson.databind.annotation.JsonNaming

@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy::class)
data class OauthToken(
    val accessToken: String = "",
    val refreshToken: String = "",
    val tokenType: String = "",
    val expiresIn: Long = 0
)
